examples/binding-patterns-in-match-statement/binding_patterns_in_match_statement.md

```diff
@@ -0,0 +1,7 @@
+# Binding patterns in `match` statement
+
+You can use the `spread` operator in a binding pattern in a `match` statement to bind the fields not explicitly mentioned in the binding pattern. This is particularly useful when working with open records.
```

Comment:
```suggestion
You can use the spread operator in a binding pattern in a `match` statement to bind the fields not explicitly mentioned in the binding pattern. This is particularly useful when working with open records.
```

See the comment re: this being a BBE for all binding patterns in a match statement rather than just the rest binding pattern.